### PR TITLE
Skip `ps_credential` test for `dsc_script` until proper test cert

### PR DESCRIPTION
**Pull request by smurawski**

### CODE REVIEW

File: spec/unit/resource/dsc_script_spec.rb [hunk 1]
```diff
@@ -71,6 +71,7 @@
     end
 
     it "has the ps_credential helper method" do
+      skip
```
Comment (thommay) [inline spec/unit/resource/dsc_script_spec.rb]: is it possible to do `skip("until we get a new cert")` the same way you can with `pending`? If so, please update with the reason. (and a date or something)

**Comment by thommay** [comment]
👍

**Comment by mwrock** [comment]
:+1:

**Comment by adamedx** [comment]
👍

**Pull request merged**